src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] Args) {
+        final int ARRAY_LENGHT = 6;
+        Figure figures[] = new Figure[6];
```

Comment:
The array declaration syntax should follow Java conventions: `Figure[] figures` instead of `Figure figures[]`.